orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandler.kt

```diff
@@ -310,7 +311,7 @@ class RunTaskHandler(
     try {
       MDC.put("stageType", type)
       MDC.put("taskType", taskModel.implementingClass)
-
+      MDC.put(AuthenticatedRequest.Header.makeCustomHeader("pipeline-description"), execution.description)
```

Comment:
you decided to use `.description` over `.name`?
In that case, since the `.description` is mostly empty, probably better to set it only when the description is nonempty

Reply:
on second thought , i think it's better to use `.name` here , also added the null check for sanity.